### ISSUE TITLE
Add more shit to the Bartender

### ISSUE THREAD

**Issue by Imagebard** [d3athrow/vgstation13#7130]
The Bartender is currently plagued by being a heavy RP role in a light RP station, his only true purpose is showing off his shotgun, making more unbroken bottles with the Still so people can stun each other, and protecting the Jukebox.

Add some shit to his drinks, re-enable Beepsky Smash stun and other shit so he isn't useless. Add some Hyperzine to his Coffee drinks. Some drinks require minerals form Miners, and those basically give nothing special in return.
At least the other RP jobs have some shit to go with them like divine assistance(Chaplain, IAA) or other fun shit(Clown, Mime) to go with them. The bartender doesn't even have his bar(Botanists and Chefs have all the access he has)

**Comment by CptWad** [comment]
I am alright so long as the doctors delight stays the only thing that could eclipse medbay
I am fine with things like regular alcohol, though some drinks should have more special powers.
Three mile island iced tea? Have it set a random block to DBC every sip
Atomic bomb? Have it be an actual explosive that you can make into an IED of some sort
Singlo? Have it give you radiation resistance while its in your system
Syndicate Bomb? Your body can't gib while its in your system (Barring divine intervention)
Manly dwarf? Drink it while you have the short power and it turns straight into doctors delight in your system
Toxins special? I think that just fucks up your insides real fucking good so just keep it doing that
Just ideas dont really suggest that they all NEED to be in

**Comment by 9600bauds** [comment]
What if we go the /tg/ way and make it so that coffee helps you up from stuns faster?

**Comment by Ephx1** [comment]
@9600bauds 
Assistants always having the jitters? I like it.

**Comment by Kurfursten** [comment]
I like having a few interesting drinks as much as the next guy, but I think bar drinks should be more fun-oriented than powergame-oriented, personally. Stuff like Gyro is really fitting for the bar, because it's a place you should go to be entertained, not as a complete substitute for medbay. More flammable drinks would be nice (the B52 is the only one currently), too.

**Comment by Exxion** [comment]
Honestly the one thing the bar needs the most in my opinion is a bottle rack
It wouldn't even be hard to code
Just a child of smartfridge

**Comment by Kurfursten** [comment]
I agree Exx. It'd be nice if that functionality could be built into the Boozeomat (as with the seed vendor) but the vendomat does not work that way sadly (it does not store items, just item types).

**Comment by Imagebard** [comment]
>I am alright so long as the doctors delight stays the only thing that could eclipse medbay

Its a mystery on why you would think this when some of the special drinks are made form plasma or other dangerous shit.
I was thinking we should leave the Doctor Delight (and Lifeline since its made out of Doctor's Delight) as the only things that heal you, and leaving other drinks as special effects like Carp Pheromones and Hyperzine, or the Radiation protection you just mentioned. And maybe a few lethal poisons, too, since Neurotoxin and Beepsky smash has shown us those are fine.

**Comment by CptWad** [comment]
@Imagebard That is exactly what I meant
eclipse meant more of a "Do better than" since the doctors delight can cure just about anything short of internal damage and clone damage
I am all for dangerous, whacky shit on the station so long as its not a large explosive (ala d20s which are too fucking big) or a quick way to make a plasma fire. I just dont want shit like redazone/peridoxon/alkasine/etc. made by the bartender. One healing drink and one healing tea is already perfect

**Comment by Imagebard** [comment]
>Went to the thread
People don't support @CptWad idea

Tough luck boy.
I guess we're going to go this way, then:
http://pastebin.com/qrP9xEZ1

**Comment by ihadtoregisterforthis** [comment]
>>the thread

**Comment by Imagebard** [comment]
The truth is that I was seduced to the !!FUN!! side and I want to intoxicate people with shotgun darts and watch them panic.
